meinberlin/apps/plans/assets/react_plans_map.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { createRoot } from 'react-dom/client'
+import { CookiesProvider } from 'react-cookie'
+import PlansListMapBox from './PlansListMapBox'
+
+function init () {
+  const plansMapBox = document.querySelectorAll('[data-map="plans"]')
+  plansMapBox.forEach(el => {
+    const projectApiUrl = el.getAttribute('data-projects-url')
+    const containersApiUrl = el.getAttribute('data-containers-url')
+    const extprojectApiUrl = el.getAttribute('data-extprojects-url')
+    const privateprojectApiUrl = el.getAttribute('data-privateprojects-url')
+    const plansApiUrl = el.getAttribute('data-plans-url')
+    const attribution = el.getAttribute('data-attribution')
+    const baseurl = el.getAttribute('data-baseurl')
+    const bounds = JSON.parse(el.getAttribute('data-bounds'))
+    const selectedDistrict = el.getAttribute('data-selected-district')
+    const selectedTopic = el.getAttribute('data-selected-topic')
+    const districts = JSON.parse(el.getAttribute('data-districts'))
+    const organisations = JSON.parse(el.getAttribute('data-organisations'))
+    const districtnames = JSON.parse(el.getAttribute('data-district-names'))
+    const topicChoices = JSON.parse(el.getAttribute('data-topic-choices'))
+    const mapboxToken = el.getAttribute('data-mapbox-token')
+    const omtToken = el.getAttribute('data-omt-token')
+    const useVectorMap = el.getAttribute('data-use_vector_map')
+    const participationChoices = JSON.parse(el.getAttribute('data-participation-choices'))
+    const root = createRoot(el)
+    root.render(
```

Comment:
You are wrapping some of the returns in `PlansListMapBox` in strict mode instead of just wrapping the whole component in strict mode here, is that because not everything works with strict mode here?

Reply:
the map cannot be wrapped, I mentioned it in the commit message but didn't add the error, basically complains that the map is already initialised

Reply:
I start to suspect that this is our first bug found/triggered by strict mode, maybe we can discuss it next week

Reply:
I also suspected that although i did find similar issues to do with how leaflet initializes the map but yes we can talk about it.